lib/doorkeeper/config.rb

```diff
@@ -159,6 +159,15 @@ def reuse_access_token
         @config.instance_variable_set(:@reuse_access_token, true)
       end
 
+      # Opt out of breaking api change to the native authorization code flow.
+      # Opting out sets the authorization code response route for native
+      # redirect uris to oauth/authorize/<code>. The default is
+      # oauth/authorize/native?code=<code>.
+      # Rationale: https://github.com/doorkeeper-gem/doorkeeper/issues/1143
+      def opt_out_native_route_change
```

Comment:
Can we rename the option to be something like `use_url_path_for_native_authorization`? To be more explicit and meaningful

Reply:
Makes sense, that's more clear, updated!